.github/workflows/build-publish-next.yml

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Build & Publish `next`
+
+on:
+  push:
+    branches:
+      - $default-branch
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        dist: [ 'alpine', 'ubi8' ]
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+      name: Checkout che-theia source code
+    - uses: actions/setup-node@v1
+      name: Configuring nodejs 12.x version
+      with:
+        node-version: '12.x'
+    - name: Login to docker.io
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+        registry: docker.io
+    - name: Login to quay.io
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+        registry: quay.io
+    - name: Build and publish images
+      run: |
+        docker image prune -a -f
+        ./build.sh --root-yarn-opts:--ignore-scripts --dockerfile:Dockerfile.${{ matrix.dist }} --push
+      env:
+        CDN_PREFIX: https://static.developers.redhat.com/che/theia_artifacts/
+        MONACO_CDN_PREFIX: https://cdn.jsdelivr.net/npm/
+    - name: Set-up npmjs auth token
+      run: printf "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}\n" >> ~/.npmrc
```

Comment:
Hello @benoitf
do you happen to know if we have `NPM_TOKEN` secret set for Che-Theia repo?

Reply:
```
curl -s -H "Authorization: token <token>" https://api.github.com/repos/eclipse/che-theia/actions/secrets
{
  "total_count": 4,
  "secrets": [
    {
      "name": "DOCKERHUB_PASSWORD",
      "created_at": "2020-12-11T14:04:09Z",
      "updated_at": "2020-12-11T14:04:09Z"
    },
    {
      "name": "DOCKERHUB_USERNAME",
      "created_at": "2020-12-11T14:04:09Z",
      "updated_at": "2020-12-11T14:04:09Z"
    },
    {
      "name": "QUAY_PASSWORD",
      "created_at": "2020-04-22T08:32:45Z",
      "updated_at": "2020-04-22T08:45:25Z"
    },
    {
      "name": "QUAY_USERNAME",
      "created_at": "2020-04-22T08:29:57Z",
      "updated_at": "2020-04-22T08:56:24Z"
    }
  ]
}
```

we don't

Reply:
As I understand, it should be [`~eclipse_che`](https://www.npmjs.com/~eclipse_che)'s token.
Do you the access to it? Can you set it for Che-Theia repo?
I remember you had some script for that.

Reply:
I've uploaded the following secrets:

CHE_NPM_AUTH_TOKEN
CHE_BOT_GITHUB_TOKEN
QUAY_ECLIPSE_CHE_USERNAME
QUAY_ECLIPSE_CHE_PASSWORD

Reply:
Great! Thanks!